.github/workflows/build-docker.yml

```diff
@@ -115,5 +118,22 @@ jobs:
             export LDFLAGS=-Wl,--no-as-needed ${LDFLAGS}
           fi
           ./scripts/ci/test-examples.sh
+      - name: Upload test results
+        # Note: upload-v3 doesn't seem to pick up the paths: delete the "false" line below once we 
```

Comment:
I assume the `always()` should also go?

Reply:
No, `always` should be there after this is working, in order to upload if the tests failed.